components/db/events.go

```diff
@@ -98,6 +100,18 @@ var _ Store = (*storeImpl)(nil)
 // Requires write-only and read-only instances for minimize conflicting writes/reads.
 // ref. https://github.com/mattn/go-sqlite3/issues/1179#issuecomment-1638083995
 func NewStore(dbRW *sql.DB, dbRO *sql.DB, tableName string, retention time.Duration) (Store, error) {
```

Comment:
Why not pass a parameter called `checkInterva`l from the outside?  Because we have to modify many files relied on the function?  Perhaps we can do like this: 
```suggestion
func NewStore(dbRW *sql.DB, dbRO *sql.DB, tableName string, retention time.Duration, checkInterval ...time.Duration) (Store, error) {
```